### DISCUSSION TITLE
Expanded Torrent Metadata

### DISCUSSION THREAD

**Discussion by ldpr** [torrust/torrust-index-gui#86] [Feature requests]
(edit @da2ce7):
- [ ] View and Edit Torrent Metadata
- [ ] Filter by Metadata Aspect
- [ ] Search by Metadata Aspect

(original @ldpr);
i understand the torrent page is getting updated, this is one thing we would find _incredibly useful_.

here's a quick mockup:

![](https://crowdfile.net/Prjmn46V.png)

I've made two quick changes here. Currently the torrent page doesn't make it clear what category the torrent is actually in. I've just tossed it in the box here but another way might be displaying it via a breadcrumb in the navbar maybe?

Secondly, we would _love_ to have a second option when uploading torrents. Categories are great, but we'd love to add a second identifier. For example, "Category: Maps, Country France". Clicking on "Maps" would just take you to the Maps Category, but clicking on _France_ would take you to all torrents with the "France" identifier/tag.

Searching works great, but it can be a little daunting to browse as there just aren't enough options to further categorize torrents.

**Comment by mickvandijke** [comment]
Hey @ldpr ,

Adding tags beside the Category option is very high on my priority list, as I fully understand the frustration of only having the Category as identifier :')

I am currently working on a new UI for the indexer and I will take your feedback about clearly showing the category and Tags for a particular torrent in mind.

Thank you for another great the suggestion!

**Comment by da2ce7** [comment]
@WarmBeer and @ldpr 
There is already a good basic schema that we might want to consider using for the Torrent Metadata:
[CreativeWork - Schema.org Type](https://schema.org/CreativeWork)

I would suggest that a torrent could be considered a CreativeWork.

**Reply by FrenchGithubUser** [reply]
I opened a discussion that this suggestion might answer to : https://github.com/torrust/torrust-index-gui/discussions/708

Just unsure if this is planned or not

**Comment by josecelano** [comment]
Hi, @ldpr @da2ce7, tags were added. Country can be handled as a tag. So, I guess the remaining features are related to improving searching. I will create new issues for the missing capabilities like:

- Search box should also use the tag. If you search for a tag, it should give you all torrents labeled with that tag.
